keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -162,6 +172,39 @@ class TitusClusterHandler(
     )
   }
 
+  fun generateContainer(container: ContainerWithDigest, account: String): Container {
```

Comment:
This allows us to try and generate an artifact on exporting a titus cluster. If the tag doesn't match any strategy we will just export the container with a digest.